src/functional/swing_slide_filter.zig

```diff
@@ -278,9 +278,9 @@ fn appendValue(comptime T: type, value: T, compressed_values: *std.ArrayList(u8)
     }
 }
 
-/// Computes the intercept coefficient of a linear function that passes through `point`
-/// with the given `slope` coefficient.
-fn computeInterceptCoefficient(slope: f80, point: Point) f80 {
+/// Computes the intercept coefficient of a linear function that passes through the `DiscretePoint`
```

Comment:
Maybe remove `DiscretePoint` from the comment for consistency as we only mention that parameter names and not the type elsewhere.

Reply:
Fixed.